crates/volta-core/src/error/details.rs

```diff
@@ -572,6 +576,11 @@ Use `npm install` or `yarn add` to select a version of {} for this project.",
 Please remove the file or pass `-f` or `--force` to override.",
                 path.display()
             ),
+            ErrorDetails::CompletionsDirCreationError { path } => write!(
+                f,
+                "Could not create directory {}.",
```

Comment:
Given that the message doesn't need anything specific to completions, we can probably re-use `CreateDirError`, which has essentially the same error message along with a permissions CTA.

Reply:
Ah, perfect – I went looking for that but had a search-fu-failure!

Reply:
Fixed in 452394c!